### DISCUSSION TITLE
Is it possible to parse windi/tailwind config object from string like below given example ?

### DISCUSSION THREAD

**Discussion by MpMeetPatel** [windicss/windicss#361] [Q&A]
The string 
```
// tailwind/windi config object

const config = {
  mode: 'jit',
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {},
  },
  variants: {
    extend: {},
  },
  plugins: [],
}
```

I want to get the config object (which is in Monaco editor as a string) as a plain JavaScript object and then want to use windicss javascript API or twind API(CSS in js tailwind library) to compile and get CSS output.

(BTW thanks for windicss - it's awesome !)

Thanks and regard,
Meet Patel

**Comment by shiftgeist** [comment]
I might be thinking to simple here, but have you tried `JSON.parse(string_config)` (given that the config object is a valid, stringified json object)?

**Reply by MpMeetPatel** [reply]
Thanks for the reply, a little bit of searching, and then I found this one 'stringify-object' npm package which is what I was looking for.
